.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Pull requests are mostly made to solve problems, add features and make other necessary adjustments.
+Every pull request should be preceded by an issue which it is supposed to solve. 
+Issue is needed before pull request so community could discuss what actually should be done about it. 
+Developers have to know if applying the request would make (breaking) changes to API/ABI (versioning purposes), what kind of test it needs (QA), etc.
+On making pull request, delete this paragraph (after reading it) and fill in following.
```

Comment:
Please add a note about coding style. The project misses a coding style document, but I'm going to add one together with the versioning policy. Just a note about following style of the existing code should be enough.